### ISSUE TITLE
Paintings override weed

### ISSUE THREAD

**Issue by Skwuruhl** [MichalD96/Perico-heist-goods-counter#18]
![Screenshot_20240718-021654](https://github.com/user-attachments/assets/dc2ac1da-cf1a-4100-9086-b2bb028e1ccc)
![Screenshot_20240718-021705](https://github.com/user-attachments/assets/d7735ec5-33f7-4726-8dfa-168ccc9fab34)

**Comment by MichalD96** [comment]
Yes, many people requested that change, to prefer bag fill over max value. I am not fan of that change either. It defeats purpose of that calculator...
